autonomy/deploy/generators/kubernetes/templates.py

```diff
@@ -306,7 +303,7 @@
 AGENT_SECRET_TEMPLATE: str = """
 apiVersion: v1
 stringData:
-    private_key: '{private_key}'
+    {ledger}_private_key.txt: '{private_key}'
```

Comment:
now we propulate the keys not assuming ethereum keys

Reply:
But why is it ending in `.txt` shouldn't it just be `{ledger}_private_key` ?

Reply:
Because the agent expects it to be private key.txt. 

If you try to run it you'll see that this creates a directory with whatever key necessary

Reply:
Feel free to provide screenshots demonstrating if this is not the case